sources/packages/forms/src/form-definitions/sfaa2024-25.json

```diff
@@ -11249,7 +11249,13 @@
                   "disabled": false,
                   "tableView": false,
                   "modalEdit": false,
-                  "defaultValue": [],
+                  "defaultValue": [
```

Comment:
Can there be at most two trusted contacts only?

Reply:
Yes, that was the existing logic.